compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/transform/DoTransform.java

```diff
@@ -249,7 +252,7 @@ public Instant timestamp() {
 
     @Override
     public PaneInfo pane() {
-      throw new UnsupportedOperationException("pane() in ProcessContext under DoTransform");
+      return PaneInfo.createPane(true, true, PaneInfo.Timing.UNKNOWN);
```

Comment:
Is this introduced to support `withWindowedWrites`? Can we perhaps do without it?

Reply:
This is because WriteFiles#expand requires `c.pane()` in the `processContext` call.